cms/toolbar/toolbar.py

```diff
@@ -329,6 +333,22 @@ def request_hook(self):
         else:
             return self._request_hook_post()
 
+    def get_cached_template(self, template):
+        if isinstance(template, Template):
+            return template
+
+        if not template in self._cached_templates:
+            self._cached_templates[template] = get_template(template)
+        return self._cached_templates[template]
+
+    @cached_property
+    def drag_item_template(self):
+        return get_template('cms/toolbar/dragitem.html')
+
+    @cached_property
+    def dragbar_template(self):
+        return get_template('cms/toolbar/dragbar.html')
+
```

Comment:
Why not just use your new `get_cached_template()` method instead of using the decorator? I think I prefer the cache to be all in one place, rather than two.